resharper/resharper-unity/src/CSharp/Daemon/Stages/Dispatcher/UnityElementProblemAnalyzer.cs

```diff
@@ -19,9 +18,15 @@ protected UnityElementProblemAnalyzer([NotNull] UnityApi unityApi)
 
         protected sealed override void Run(T element, ElementProblemAnalyzerData data, IHighlightingConsumer consumer)
         {
+            // TODO: Check other process kinds. Should we run for everything?
```

Comment:
@krasnotsvetov can you verify what kinds we should be ignoring here? We obviously want to work for visible document and SWEA. These changes require `OTHER` to quickly find highlightings to fix, but what about the other values?

Reply:
I am not sure, that we should ignore `INCREMENTAL_SOLUTION_ANALYSIS`, but ignoring GLOBAL_WARNINGS is ok. Only performance critical inspections (and burst in future) will use global stage